src/assets/style/modules/button.scss

```diff
@@ -9,18 +9,22 @@
   cursor: pointer;
   border-radius: 5px;
   border: 2px solid $primary;
+
   &--primary {
     background: $primary;
+
     &:hover {
       background: $primaryh;
       border: 2px solid $primaryh;
     }
+
     &:active {
       background: $primarya;
       border: 2px solid $primarya;
```

Comment:
`primarya` and `primaryh` are confusing for me? What about rest of viewers? Are you understand these names?

Reply:
This is my naming of these properties. h-hover a-active. We have the last moment to standardize the names. Please create a standardization for the naming of global variables on clickup